src/app/App.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Chart from "../components/Chart";
+
+function App() {
+  return (
+    <div>
+      <Chart />
+    </div>
```

Comment:
```suggestion
    <>
      <Chart />
    </>
```
밑에 웅민님 말씀처럼 fragment 를 활용해도 좋을 거 같습니다 ~~!